src/app/components/App.tsx

```diff
@@ -23,6 +23,7 @@ import {
 import pkg from '../../../package.json';
 
 import './styles/app.scss';
+import ContextualHelpHandler from '../../help/components/contextual/ContextualHelpHandler';
```

Comment:
Should we wrap that in a lazy loader like the rest of the routes? Since it takes a function that returns a promise, we could make it wait a bit in order to prioritise the rest of the page and content. Something like
`lazy(() => sleep(1000).then(() => import(...)))`